src/mapper/map.rs

```diff
@@ -1,19 +1,20 @@
 use std::collections::HashMap;
 
-use pallas::codec::minicbor::bytes::ByteVec;
-use pallas::crypto::hash::Hash;
+use pallas::codec::minicbor::{self, bytes::ByteVec};
+use pallas::crypto::hash::{Hash, Hasher};
 use pallas::ledger::primitives::alonzo::{
     self as alonzo, AuxiliaryData, Block, Certificate, InstantaneousRewardSource,
-    InstantaneousRewardTarget, Metadatum, Relay, TransactionInput, TransactionOutput, Value,
+    InstantaneousRewardTarget, Metadatum, NativeScript as PallasNativeScript, Relay,
```

Comment:
I'm not completely satisfied with the `NativeScript` name conflict and `PallasNativeScript` alias name but I have not found a more satisfying solution yet. I'm open to suggestions.

Reply:
Yep, type name collisions are pretty common when mapping between models. One way I found helpful is to qualify the types with the module. For example, `alonzo::NativeScript`. It adds some extra characters, but easier to follow in the code IMO.

In this particular case, `PallasNativeScript` doesn't seem too bad. Whichever you prefer is fine by me. Your call.

Reply:
Good idea. I prefer `alonzo::NativeScript` too. I will rework the patchset with it.

Reply:
The new version uses alonzo::NativeScript, I'm resolving this conversation.